server/stream.go

```diff
@@ -80,6 +80,9 @@ type StreamConfig struct {
 	// AllowRollup allows messages to be placed into the system and purge
 	// all older messages using a special msg header.
 	AllowRollup bool `json:"allow_rollup_hdrs"`
+
+	// Additional metadata for the Stream.
```

Comment:
```suggestion
	// Metadata is additional metadata for the Stream.
```

Reply:
ah, good point. Fixed both.

We should do a sweep through comments and enable lint for that.